src/modules/bookings/components/bookings-navbar.tsx

```diff
@@ -0,0 +1,35 @@
+import { capitalizer } from '#/shared/helper/capitalizer.helper';
+import { BookingStatus } from '../constants/booking-status.constant';
+import Typography from '@mui/material/Typography';
+
+interface BookingsNavbarProps {
+  selectedCategory: BookingStatus;
+  onCategoryChange: (status: BookingStatus) => void;
+}
+
+const styles = {
+  navItem: (isSelected: boolean) => ({
+    width: '100%',
+    textAlign: 'center',
+    borderBottom: isSelected ? '3px solid #000' : '1px solid #e9e9e9',
```

Comment:
Maybe it is better to make border bottom pink

Reply:
> Maybe it is better to make border bottom pink

Agreed. Style consistency

Reply:
Fixed